include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -2008,6 +2008,8 @@ namespace internal
       , shape_gradients(nullptr)
       , shape_hessians(nullptr)
     {
+      (void)n_rows;
```

Comment:
Shouldn't we initial the class fields here?

Reply:
Correct; to my defense, I did not write the code :wink: